src/modules/contacts/components/SearchBox.vue

```diff
@@ -39,12 +27,9 @@ const props = defineProps<{
   modelValue: string;
 }>();
 
-const basicSearchValue = ref(props.modelValue);
-watch(
-  () => props.modelValue,
-  (value) => {
-    basicSearchValue.value = value;
-  }
-);
+const searchText = computed({
+  get: () => props.modelValue,
+  set: (text) => emit('update:modelValue', text),
+});
```

Comment:
Would love to talk about `watch()` v `computed()` – I remember reading about it, I'd be curious to hear why you changed this.

Reply:
In this case it's just because I moved the `watch()` into `AppSearchInput`, so this component now just proxies the `modelValue` using a `computed()`